data/tutorials/getting-started/3_01_ocaml_on_windows.md

```diff
@@ -6,8 +6,211 @@ description: >
 category: "Resources"
 ---
 
-There is a [DkML] Windows
-installer that we recommend for new users. However, while [DkML] has a modern OCaml 4.14.0 compiler,
+## opam
+
+Opam now features a fully native Windows compatible installation process that we
+recommend new users to follow below.
+
+In order to have access to OCaml on a Windows PC, users have to use opam so that's our
+first step.
+
+Windows Developers out there will already be familiar with `winget` which is
+the default package installer for Windows. Opam is now distributed with it. 
+
+```shell-session
+$ winget show OCaml.opam
+Found opam [OCaml.opam]
+Version : 2.2.1
+Publisher : OCaml, Inc.
+[...]
+```
+
+Installing opam is done by running the following command in your terminal:
+
+```shell-session
+> winget install OCaml.opam
+Found opam [OCaml.opam] Version 2.2.1
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://github.com/ocaml/opam/releases/download/2.2.1/opam-2.2.1-x86_64-windows.exe
+  ██████████████████████████████  9.89 MB / 9.89 MB
+Successfully verified installer hash
+Starting package install...
+Path environment variable modified; restart your shell to use the new value.
+Command line alias added: "opam"
+Successfully installed
+Notes: See https://opam.ocaml.org/blog for the latest news on how to use opam.
+```
+
+Once the package is installed, you can launch a new shell to have access to
+your fresh opam binary.
+
+```shell-session
+$ opam --version
+2.2.1
+```
+
+As for any other platform out there, you have to initialise your opam
+installation with the `opam init` command. 
+
+The first thing to know is that opam requires a UNIX-like environment to
+function. By default, opam relies on `cygwin` and is also compatible with
+`msys2`.
+
+At *init-time*, opam scans your machine for available UNIX environments and
+prompts you to choose your favourite option. That being said, it is recommended
+to let it create its own internal Cygwin installation that will remain managed
+by opam. This allows to cut down the possible interferences that other tools
+that interact with such environments might introduce. Think of it as a
+sandboxed environment.
+
+You will notice that the repository information fetching stage takes a while to
+complete. This is normal and related to how files are handled on Windows, we
```

Comment:
It is a bit weird to be assigning blame in a tutorial, but especially blaming Windows to Windows users is weirder still. (Also a bit inaccurate ... I would say / have said that the root cause is thinking that littering files on a filesystem is an adequate replacement for databases and package indexes, a line of thinking that most other package managers reject. Akin to blaming the performance without considering the data structure. But the broader point is all opinions are argumentative with no net benefit)